src/robot/devices/core/lsm6/lsm6.ts

```diff
@@ -529,13 +549,13 @@ export default class LSM6 {
             if (shouldProcess) {
                 switch (i % 6) {
                     case 0:
-                        tempFrame.gyroX = ((val * gyroScaleFactor) / 1000) - this._gyroOffset.x;
+                        tempFrame.gyroX = ((val * gyroScaleFactor) / 1000) - this._gyroOffset.x + this._gyroAdditionalOffset.x;
```

Comment:
Why add additional offset instead of subtract like original offset from calibration?

Reply:
It's mainly to allow on-the-fly adjustment in robot code (if necessary) and act in conjunction with the pre-calibrated zero offset (and not replace it)

Reply:
Could go either way. The zero offsets were all positive, hence the subtraction.